### PR TITLE
Remove deprecated botocore.requests library 

### PR DESCRIPTION
**Pull request by robinkwilson**
- Replaced botocore.requests with urllib3

### CODE REVIEW

File: cloudformation/stack.yaml [hunk 1]
```diff
@@ -1409,8 +1409,9 @@ Resources:
         ZipFile: |
           import json
           import boto3
-          from botocore.vendored import requests
+          import urllib3
           from botocore.exceptions import ClientError
```
Comment (johnshaughnessy) [inline cloudformation/stack.yaml:1413]: Are `ClientError`s from `botocore.exceptions` and other `botocore` bits still expected to be used? (Sorry if this is a dumb question -- Is there something I can read to catch up on the `botocore` deprecation issue?)

File: cloudformation/stack.yaml [hunk 2]
```diff
@@ -1409,8 +1409,9 @@ Resources:
         ZipFile: |
           import json
           import boto3
-          from botocore.vendored import requests
+          import urllib3
```
Comment (johnshaughnessy) [inline cloudformation/stack.yaml:1412]: Where do these dependencies come from? Does something else run `pip install urllib3`?

**Review by johnshaughnessy — COMMENTED** [review]
The new `request` look fine to me after glancing its [docs](https://urllib3.readthedocs.io/en/latest/reference/urllib3.request.html#urllib3.request.RequestMethods.request). Left a couple questions because I'm not familiar with this python code.

File: cloudformation/stack.yaml (hunk 1 above)
Comment (robinkwilson) [reply cloudformation/stack.yaml:1413]: All good. I had this same question and asked an aws support person yesterday, luckily only the botocore.requests library is being deprecated, this one is still functional.

File: cloudformation/stack.yaml (hunk 2 above)
Comment (robinkwilson) [reply cloudformation/stack.yaml:1412]: Also good question. Lambda functions come with this installed. So we're good here too. :)

**Review by johnshaughnessy — APPROVED** [review]

**Pull request merged**